src/main/java/ser/mil/rideapp/controller/request/RideRequest.java

```diff
@@ -1,4 +1,7 @@
 package ser.mil.rideapp.controller.request;
 
-public record RideRequest(double startLat, double startLong, double endLat, double endLong, String customer) {
+import ser.mil.rideapp.domain.model.Currency;
+
+public record RideRequest(double startLat, double startLong, double endLat, double endLong, String customer,
+                          Currency baseCurrency, Currency finalCurrency) {
```

Comment:
Jesli masz baseCurrency to trzeba to rowniez obsluzyc w PricingService, czyli umozliwic wyliczanie ceny dystansu dowolnej waluty na dowolną, nie tylko polskiej na dowolną